src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -444,14 +433,15 @@ public String getElementAttribute(String attributeLocator) {
             "Example: ${id}=    Get Element Attribute   css:h1  id\r\n" +
             "\r\n" +
             "Passing attribute name as part of the locator was removed in SeleniumLibrary 3.2. The explicit attribute argument should be used instead.")
-    @ArgumentNames({ "locator", "attribute" })
-    public String getElementAttribute(String locator, String attribute) {
+    @ArgumentNames({ "locator", "attribute=None" })
+    public String getElementAttribute(String locator, String... attribute) {
+        String attributeName= robot.getParamsValue(attribute, 0, "None");
```

Comment:
String attributeName = -formating

Reply:
What do you mean exactly?

Reply:
String attributeName = robot.getParamsValue(attribute, 0, "None");
                                 ^

Reply:
ah found it, space between attributeName and = sign. 
And fixed...